release.dockerfile

```diff
@@ -16,6 +16,7 @@ RUN chmod 777 /resources/docker-entrypoint.sh
 
 EXPOSE 8080
 ENV PYTHONUNBUFFERED True
+ENV NO_NETIFACES=1
```

Comment:
@rodja I guess this should be set to "true".

Reply:
Oh yes. I forgot!